nixosModules/common.nix

```diff
@@ -42,51 +25,4 @@ in {
   nix.settings.trusted-users = with config.users; [
     users.root.name
   ];
-
```

Comment:
This is going in the wrong direction.

We should have `steward`, `benefice` and `drawbridge` modules moved to this directory and addressable as e.g. `nixosModules.benefice`

Most importantly, this change breaks https://github.com/profianinc/images and severely limits usability of this repository for no reason